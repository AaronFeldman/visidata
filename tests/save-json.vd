sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/test.jsonl	o	
test			save-sheet	tests/golden/save-json.csv	^S	
test			save-sheet	tests/golden/save-json.json	^S	
test			save-sheet	tests/golden/save-json.md	^S	
test			save-sheet	tests/golden/save-json.html	^S	
test			save-sheet	tests/golden/save-json.txt	^S	
