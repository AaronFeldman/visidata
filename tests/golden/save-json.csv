key1,qty,key2,amt
2016-01-01 11:00:00,1,foo,
2016-01-01 1:00,2,,3
,4,baz,43.2
"{""key1"": ""2017-12-25 8:43"", ""foo"", ""amt"": "".03"", ""qty"": null}
","{""key1"": ""2017-12-25 8:43"", ""foo"", ""amt"": "".03"", ""qty"": null}
","{""key1"": ""2017-12-25 8:43"", ""foo"", ""amt"": "".03"", ""qty"": null}
","{""key1"": ""2017-12-25 8:43"", ""foo"", ""amt"": "".03"", ""qty"": null}
"
2017-12-25 8:44,16,bar,.3
,32,baz,3.3
2018-07-27 4:44,64,,9.1
2018-07-27 16:44,,bar,
2018-07-27 18:44,256,baz,.01
2018-10-20 18:44,30.0,foo,.01
